apps/CA.pl.in

```diff
@@ -131,9 +131,9 @@ if ($WHAT eq '-newcert' ) {
     # create a certificate request
     $RET = run("$REQ -new -keyout $NEWKEY -out $NEWREQ $DAYS $EXTRA{req}");
     print "Request is in $NEWREQ, private key is in $NEWKEY\n" if $RET == 0;
-} elsif ($WHAT eq '-newreq-nodes' ) {
+} elsif ($WHAT eq /-newreq(\-nodes)?/ ) {
```

Comment:
No-no-no, suggestion was to *remove* *this* elsif and modify the preceding one. Well, now you can simply remove the preceding elsif. But either way modifying just two lines was not actual suggestion. Also note that it renders CHANGES entry misleading. I mean it reads "removed -newreq-nodes" while what happens here is "reimplemented -newreq-nodes".

However! It's not actually given that it makes sense to keep it. I mean even if reimplementation does it, we still might find it appropriate to omit it. Or at least to declare deprecated. In which case it would be appropriate to issue warning that reads "stop using this, use \<this\> instead."

Reply:
And there is typo, it's not "`$WHAT eq ...`" but "`$WHAT =~ ...`"

Reply:
And it was `/\-new...`, not `/-new...`. Well, `/-new.../` works, but it looks ambiguous and hence less readable.